src/components/filters/component.jsx

```diff
@@ -42,7 +42,11 @@ const FiltersComponent = ({
           <Dropdown
             className={styles.dropdown}
             title={
-              scenario && parsedScenarios && parsedScenarios.find(s => s.value === scenario).label
+              `Future scenarios: ${scenario &&
+                parsedScenarios &&
+                parsedScenarios
+                  .find(s => s.value === scenario)
+                  .label.replace(/^[^-]*[^ -]*- */g, '')}` // split by hyphen, get only 'RCP' part
```

Comment:
Since we have the short name in the data table, you don't need to extract it this way - just pass `shortName` to the `parsedScenarios` object (in `pages/bioclimatic/index.js`)